### PR TITLE
Standardise margin on headers

### PR DESCRIPTION
**Pull request by titlescreen**
Some questions had margins and others didn't so they were displaying
inconsistently. Also included labels for questions that didn't have them

### CODE REVIEW

File: app/views/contact-method.html [hunk 1]
```diff
@@ -24,7 +24,10 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form method="POST" action="check-answers">
-          <h1 class="govuk-heading-xl">Email Address</h1>
+          <h1 class="govuk-label-wrapper" id="emailQuestion"><label class="govuk-label govuk-label--xl" for="emailQuestion">
```
Comment (pezholio) [inline app/views/contact-method.html]: Is it OK to put the opening `label` tag on the next line? 

Just helps readabilty for when we move everything over tho a Rails app 👍

File: app/views/eligible-confirmed.html [hunk 2]
```diff
@@ -23,8 +23,9 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">
+      <h1 class="govuk-label-wrapper" id="eligibleTitle"><label class="govuk-label govuk-label--xl" for="eligibleTitle">
```
Comment (pezholio) [inline app/views/eligible-confirmed.html]: Same here

File: app/views/private-agency.html [hunk 3]
```diff
@@ -23,7 +23,10 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">Are you employed directly by your school?</h1>
+      <h1 class="govuk-label-wrapper" id="agencyQuestion"><label class="govuk-label govuk-label--xl" for="agencyQuestion">
```
Comment (pezholio) [inline app/views/private-agency.html]: Same thing with the linebreak here

File: app/views/school.html [hunk 4]
```diff
@@ -23,8 +23,9 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">
-        Which school do you teach at?
+      <h1 class="govuk-label-wrapper" id="schoolQuestion"><label class="govuk-label govuk-label--xl" for="schoolQuestion">
```
Comment (pezholio) [inline app/views/school.html]: Ditto

File: app/views/contact-method.html (hunk 1 above)
Comment (pezholio) [reply app/views/contact-method.html]: Also, does `emailQuestion` refer to anything in the rendered page?  I could be wrong, but I thought `for` had to refer to an `input` field in a label?

Comment (titlescreen) [reply app/views/contact-method.html]: 634695f should resolve this

Comment (titlescreen) [reply app/views/contact-method.html]: I think a for can link to anything that has an `id` I think in the past IO've used them for aria descriptions on images or movies but it was totally redundant having it reference the direct parent.

**Pull request merged**